cns/service.go

```diff
@@ -316,3 +327,20 @@ func (service *Service) SendErrorResponse(w http.ResponseWriter, errMsg error) {
 	err := acn.Encode(w, &resp)
 	logger.Errorf("[%s] %+v %s.", service.Name, &resp, err.Error())
 }
+
+// TLSVersionNumber returns the version number for the provided TLS version name
+// (e.g. 0x0301)
+func TLSVersionNumber(versionName string) (uint16, error) {
```

Comment:
I am surprised the standard library doesn't have a parse function for this. I would change this to `parseTLSVersionNumber` since that's the convention you would find there.